README.md

```diff
@@ -28,6 +29,22 @@ sudo dpkg -i omnect-cli_<version>_amd64.deb
 ```
 **Note**: `dpkg` lists necessary runtime dependencies in case they are not present.
 
+## Docker image
+
+`omnect-cli` is also provided as docker image.<br>
+
+Example usage:
+
+```ssh
+docker run --rm -it \
+  --mount type=bind,source="$(pwd)"/,target=/source \
```

Comment:
`-v $(pwd):/source`  should behave equally when using a local dir:
was there a reason to choose the more complicated parameterization?